msal/token_cache.py

```diff
@@ -150,31 +150,6 @@ def __add(self, event, now=None):
         with self._lock:
             now = int(time.time() if now is None else now)
 
-            if access_token:
-                expires_in = int(  # AADv1-like endpoint returns a string
-			response.get("expires_in", 3599))
-                ext_expires_in = int(  # AADv1-like endpoint returns a string
-			response.get("ext_expires_in", expires_in))
-                at = {
-                    "credential_type": self.CredentialType.ACCESS_TOKEN,
-                    "secret": access_token,
-                    "home_account_id": home_account_id,
-                    "environment": environment,
-                    "client_id": event.get("client_id"),
-                    "target": target,
-                    "realm": realm,
-                    "token_type": response.get("token_type", "Bearer"),
-                    "cached_at": str(now),  # Schema defines it as a string
-                    "expires_on": str(now + expires_in),  # Same here
-                    "extended_expires_on": str(now + ext_expires_in)  # Same here
-                    }
-                if data.get("key_id"):  # It happens in SSH-cert or POP scenario
-                    at["key_id"] = data.get("key_id")
-                if "refresh_in" in response:
-                    refresh_in = response["refresh_in"]  # It is an integer
-                    at["refresh_on"] = str(now + refresh_in)  # Schema wants a string
-                self.modify(self.CredentialType.ACCESS_TOKEN, at, at)
-
             if client_info and not event.get("skip_account_creation"):
                 account = {
```

Comment:
The order of adding tokens to the cache matters. Always create the `account` first so that old `account` can be removed first.